zilliqa/src/consensus.rs

```diff
@@ -434,9 +442,12 @@ impl Consensus {
                     )));
                 };
             } else {
-                self.reset_timeout.send(Duration::from_millis(
-                    empty_block_timeout_ms - time_since_last_view_change + 1,
-                ))?;
+                self.reset_timeout.send(
+                    self.config
+                        .consensus
+                        .empty_block_timeout
```

Comment:
is this missing a `.as_millis()` as on line 425

Reply:
It sends and receives a `Duration` so I don't think that's necessary.